README.md

```diff
@@ -46,9 +46,9 @@ Notice:  Constant BBBB does not exist in self scope in fixtures/simple/undefined
 
     return self::BBBB; 
 
-Notice:  You are trying to cast 'string' to 'string'. in fixtures/simple/code-smell/StandardFunctionCall.php on 16 [stupid.cast]
+Notice:  You are trying to cast 'string' to 'string' in fixtures/simple/code-smell/StandardFunctionCall.php on 16 [stupid.cast]
```

Comment:
This message is generated by PHPSA, we should also fix it [in the analyzer](https://github.com/ovr/phpsa/blob/49827aed3dc3b425b862ea99762953c42cd70fc5/src/Analyzer/Pass/Expression/Casts.php#L50)